test/test_ops.py

```diff
@@ -456,6 +457,19 @@ def get_sample():
                         check_alias_annotation(name, (get_sample(),) + sample.args, sample.kwargs,
                                                func_type=func_type, aten_name=op.aten_name)
 
+                        # TODO: use script graph as well
+                        checked_shape_analysis = False
+                        if supports_tracing:
+                            out = variant(get_sample(),
+                                                      *sample.args,
+                                                      **sample.kwargs)
+
+                            if isinstance(out, torch.Tensor):
```

Comment:
Probably want a TODO here to handle ops that output multiple tensors?

Reply:
will do, there also isn't support for shape analysis functions with multiple outputs either..